superbench/runner/runner.py

```diff
@@ -22,21 +22,23 @@
 
 class SuperBenchRunner():
     """SuperBench runner class."""
-    def __init__(self, sb_config, docker_config, ansible_config, sb_output_dir):
+    def __init__(self, sb_config, docker_config, ansible_config, sb_output_dir, log_flushing=False):
```

Comment:
why not log_flushing is not in sb_config? I find that it's strange for log_flushing is equal to sb_config, docker_config, etc.

Reply:
updated